tests/interfaces/test_jacobian_products.py

```diff
@@ -225,6 +225,8 @@ def test_execute_jvp_basic(self, jpc, shots):
         """Test execute_and_compute_jvp for a simple single input single output."""
         if shots and not _accepts_finite_shots(jpc):
             pytest.skip("jpc does not work with finite shots.")
+        if isinstance(jpc, DeviceJacobianProducts) and "lightning" in jpc._device.name:
+            pytest.skip("Lightning devices don't have JVP method")
```

Comment:
```suggestion
            pytest.xfail("Lightning devices don't have JVP method")
```
? Maybe.  Can't hurt and will let us know if things change.